python/edgar_filing_searcher/parsers/main.py

```diff
@@ -17,6 +18,17 @@ def create_url_list(url_edgar_current_events):
 
 def send_data_to_db(company_row, edgar_filing_row, data_13f_table):
     """This function sends data to the database"""
+    company_from_database = Company.query.filter_by(cik_no=company_row.cik_no).first()
+    filing_from_database = EdgarFiling.query\
+        .filter_by(accession_no=edgar_filing_row.accession_no).first()
```

Comment:
Instead of filing_from_database, filing_in_database. Also, it needs to be a boolean for this to make sense.